pymeasure/instruments/instrument.py

```diff
@@ -84,7 +84,7 @@ def __get__(self, obj, objtype=None):
 
     def __set__(self, obj, value):
         if self.fset is None:
-            raise AttributeError("Can't set attribute")
+            raise AttributeError(f'Can\'t set attribute {self.name}')
```

Comment:
` f"Can't set attribute {self.name}"` does not work? (Or `cannot`, for that matter)

Reply:
Change done. Thanks